src/swap/SwapScreen.tsx

```diff
@@ -989,13 +1017,11 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   fromSwapAmountInput: {
-    borderBottomWidth: 0,
-    borderTopLeftRadius: 16,
-    borderTopRightRadius: 16,
+    zIndex: -1,
+    marginBottom: Spacing.Smallest8,
   },
   toSwapAmountInput: {
-    borderBottomLeftRadius: 16,
-    borderBottomRightRadius: 16,
+    zIndex: -1,
```

Comment:
we should minimise changes to the zIndex because it makes the layout unpredictable. i think you can remove all the zIndex styles and apply a single `zIndex: 1` to `switchTokensContainer`. at least this minimises the places that we manually set the layout order